infra/terraform/platform/main.tf

```diff
@@ -226,6 +226,18 @@ module "kubernetes_master_monitoring" {
   ), var.tags)}"
 }
 
+module "kubernetes_master_asg_monitoring" {
+  source = "../modules/asg_cloudwatch_alerts"
+
+  name          = "${terraform.workspace}-kubernetes-master-asg-alerts"
+  asg_names     = ["master-eu-west-1a.masters.${terraform.workspace}.mojanalytics.xyz", "master-eu-west-1b.masters.${terraform.workspace}.mojanalytics.xyz", "master-eu-west-1c.masters.${terraform.workspace}.mojanalytics.xyz"]
```

Comment:
This is coming from Kops, that's why we don't have this list in Terraform, right? 👌

Reply:
Yeah that's right, we could maybe read from a data source but probably isn't worth it.

Reply:
Agreed ☺️ 👌 - not ideal but at least it's simple ☺️